cachito/web/api_v1.py

```diff
@@ -77,3 +111,20 @@ def create_request():
 @api_v1.errorhandler(ValidationError)
 def handle_validation_error(e):
     return flask.jsonify(error=str(e)), 400
+
+
+def _get_request_obj(request_id):
+    """
+    Retrieve request object for given ID
+
+    :param int request_id: the value of the request ID
+    :return: a Request
+    :rtype: cachito.models.Request
+    :raise NotFound: if the request is not found
+    :raise ValidationError: if the request ID is of invalid format
+    """
+    try:
+        request_id = int(request_id)
```

Comment:
Instead, you could just set the route to:
```python
@api_v1.route('/requests/<int:request_id>', methods=['GET'])
```

Reference: http://flask.pocoo.org/docs/1.0/quickstart/#variable-rules

Reply:
To get the right type? Yeah, cool hint. I'll try it out!